CanHazFunny/CanHazFunny/JsonJokeService.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Text.Json;
+using System.Net.Http;
+
+namespace CanHazFunny
+{
+    public class JsonJokeService : JokeService
+    {
+
+        new public string? GetJoke()
+        {
+            _Joke? joke = JsonSerializer.Deserialize<_Joke?>(HttpClient.GetStringAsync(URL).Result);
+            return joke?.joke;
+        }
+
+
+        private class _Joke
+        {
+            public string? joke { get; set; }
+        }
```

Comment:
I would move this class to be outside of the JsonJokeService class. Aside from that, `joke` should be pascal case. The issue of course, is it doesn't deserialize properly that way.

To fix that you could import `Newtonsoft.Json` and do the following...
```suggestion
        private class _Joke
        {
            [JsonProperty("joke")]
            public string? Joke { get; set; }
        }
```